packages/components/tests/dummy/app/styles/components/dummy-toc.scss

```
@@ -0,0 +1,22 @@
+.dummy-aside-list {
+  box-sizing: border-box;
+  list-style: none;
+  padding: 0;
+  margin: auto;
+}
+
+.dummy-aside-list-item {
+  box-sizing: border-box;
+  width: 100%;
+  a {
+    border: 2px solid transparent;
+    box-sizing: border-box;
+    padding: 0.5rem;
+    width: 100%;
+    display: flex;
+    &:hover {
+      background-color: aliceblue;
+      border-left-color: blue;
+    }
```

```suggestion
      border-top-right-radius: 5px;
      border-bottom-right-radius: 5px;
    }
```

Added a subtle design polish suggestion. Eventually would love to see us sort out a way to do the border-radius on all sides, but the use of `border-left-color` makes that challenging and I don't want to make this implementation more complicated right now. 

This is all good with me if we get @didoo's suggestions in place.